### PR TITLE
Fix u14 cython

### PR DESCRIPTION
**Pull request by Baaaaam**
This supposedly fix the finding issue with ubuntu 14.04 with python2

### CODE REVIEW

File: cmake/FindJinja2.cmake [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+# distributed under the Apache License with the above modification is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the Apache License for the specific
+# language governing permissions and limitations under the Apache License.
+#
+# - Find Modo libraries
+#
+# Jinja2 is a python library, ensure that it is available for use with our
+# specified version of Python.
+#
```
Comment (scopatz) [inline cmake/FindJinja2.cmake]: I don't think we need to copy this if...

File: cmake/FindJinja2.cmake [hunk 2]
```diff
@@ -1,12 +1,44 @@
...
+endif()
+
+execute_process(
+  COMMAND 
+  "${PYTHON_EXECUTABLE}" "-c" "import jinja2"
+  RESULT_VARIABLE
+  jinja2ImportResult 
+  )
+if (jinja2ImportResult EQUAL 0)
+  set(JINJA2_FOUND True)
```
Comment (scopatz) [inline cmake/FindJinja2.cmake]: I think the previous version was better.

File: cmake/FindJinja2.cmake [hunk 3]
```diff
@@ -1,12 +1,44 @@
...
+#
+if (NOT PYTHON_EXECUTABLE)
+  return()
+endif()
+
+execute_process(
+  COMMAND 
+  "${PYTHON_EXECUTABLE}" "-c" "import jinja2"
+  RESULT_VARIABLE
+  jinja2ImportResult
```
Comment (scopatz) [inline cmake/FindJinja2.cmake]: This does not follow the regular naming conventions

**Comment by Baaaaam** [comment]
@scopatz do you like it better ?

`find_package_handle_standard_args(Jinja2 REQUIRED_VARS JINJA2_FOUND)` was not working on ubuntu 14.04 +python2, 

getting and returning the version number seems to work...
I ma not sure what is the difference between `REQUIRED_VARS` and `DEFAULT_MSG`, I used FindNumpy.cmake as a model...

File: cmake/FindJinja2.cmake [hunk 4]
```diff
@@ -1,12 +1,16 @@
...
-if(JINJA2_RTN MATCHES 0)
-    set(JINJA2_FOUND 1)
-else()
-    set(JINJA2_FOUND 0)
-endif()
+
+find_package(PythonInterp)
+
+IF(PYTHONINTERP_FOUND)
+  EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} "-c" "import jinja2;print(jinja2.__version__)"
```
Comment (scopatz) [inline cmake/FindJinja2.cmake]: PEP8 Space after semicolon

**Pull request merged**